### PR TITLE
Lh ycx patch 1

### PR DESCRIPTION
**Pull request by lh-ycx**
This patch contains the modification described in #9

### CODE REVIEW

File: tools/pt/Makefile [hunk 1]
```diff
@@ -12,7 +12,10 @@ obj/libxed.a:
 	./xed/mfile.py
 
 build/bin/ptxed:
-	cmake -DPTDUMP=ON -DPTXED=ON -DPTTC=ON -DSIDEBAND=ON -DPEVENT=ON -DFEATURE_ELF=ON -DFEATURE_THREADS=ON -DXED_INCLUDE=./obj/wkit/include/xed -DXED_LIBDIR=./obj/ -B build ./libipt/
+	mkdir -p build
+	cd build && \
+	cmake -DPTDUMP=ON -DPTXED=ON -DPTTC=ON -DSIDEBAND=ON -DPEVENT=ON -DFEATURE_ELF=ON -DFEATURE_THREADS=ON -DXED_INCLUDE=./obj/wkit/include/xed -DXED_LIBDIR=./obj/ ../libipt/ && \
```
Comment (carter-yagemann) [inline tools/pt/Makefile:17]: Now that you're building inside `pt/build`, the relative paths in the cmake command need to be adjusted:

```suggestion
	cmake -DPTDUMP=ON -DPTXED=ON -DPTTC=ON -DSIDEBAND=ON -DPEVENT=ON -DFEATURE_ELF=ON -DFEATURE_THREADS=ON -DXED_INCLUDE=../obj/wkit/include/xed -DXED_LIBDIR=../obj/ ../libipt/ && \
```

File: tools/pt/libipt/ptxed/CMakeLists.txt [hunk 2]
```diff
@@ -30,6 +30,7 @@ set(XED_LIBDIR  "" CACHE PATH "")
 include_directories(
   include
   ../libipt/internal/include
+  ../../obj/wkit/include/xed
```
Comment (carter-yagemann) [inline tools/pt/libipt/ptxed/CMakeLists.txt:33]: This line is no longer needed once you fix the relative paths in `tools/pt/Makefile`

```suggestion
```

File: tools/pt/libipt/ptxed/CMakeLists.txt [hunk 3]
```diff
@@ -38,6 +39,7 @@ include_directories(SYSTEM
 
 link_directories(
   ${XED_LIBDIR}
+  ../../obj/wkit/lib
```
Comment (carter-yagemann) [inline tools/pt/libipt/ptxed/CMakeLists.txt:42]: This line is no longer needed once you fix the relative paths in `tools/pt/Makefile`

```suggestion
```

**Review by carter-yagemann — CHANGES_REQUESTED** [review]
Hi @lh-ycx, thanks for your PR. Please see my review comments and retest on your end.

**Comment by lh-ycx** [comment]
Thanks for the suggestions. The revision goes well! I think it is ready to merge.

**Review by carter-yagemann — APPROVED** [review]

**Pull request merged**